### ISSUE TITLE
Can I update my device without jailbreak 

### ISSUE THREAD

**Issue by villager1314** [pwned4ever/Th0r_Freya#36]

**Comment by pwned4ever** [comment]
I'm lost what you mean.

Sent from my iPhone

On Feb 11, 2023, at 3:42 AM, villager1314 ***@***.***> wrote:

﻿

**Comment by villager1314** [comment]
I typed the wrong.Can I update my device without restore rootFS

**Comment by pwned4ever** [comment]
No, you must restorerootfs before doing any updates, then you can update the os version.

Sent from my iPhone

On Feb 17, 2023, at 5:31 AM, villager1314 ***@***.***> wrote:

﻿

I typed the wrong.Can I update my device without restore rootFS
